lightstep/resource_dashboard.go

```diff
@@ -9,6 +9,49 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 )
 
+func getThresholdSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"color": {
+			Type:     schema.TypeString,
+			Required: true,
+			ValidateFunc: validation.StringInSlice([]string{
+				"#AA3018",
+				"#B67D0C",
+				"#3C864F",
+				"#1B7BBB",
+				"#DC7847",
+				"#78469B",
+				"#37A2AE",
+				"#B03B7F",
+				"#1F40C1",
+				"#8B7255",
+				"#826CEF",
+				"#D56DD5",
+				"#6699CC",
+			}, false),
```

Comment:
wait i thought we were going to allow people to pass anything they wanted here?
```suggestion
```

Reply:
Currently, it's a dropdown of options, but I think there was talk of adding a color wheel at some point, so I can remove this

Reply:
Yup, it's currently limited

Reply:
we _want_ to limit the allowed values in terraform? why? is the concern just that we wouldn't know how to show a custom color in the chart config item's list? that seems like a minor concern compared to not being able to configure the color i want

Reply:
removed